### DISCUSSION TITLE
How to use DJL in Android app for digits recognition model to add On-device training functionality.

### DISCUSSION THREAD

**Discussion by Ashok-Kumar-dharanikota** [deepjavalibrary/djl#2563] [Development]
I want to create an On-device training functionality for a mobile app to reTrain the pretrained digits recognition model. I tried tensorflow lite, Deeplearning4j. But I am unable to achieve it 😭. Can anyone help me, how to do this using DJL. 

I really appreciate your help in this.

**Comment by frankfliu** [comment]
Currently, we don't support training on Android. We didn't see strong demand for android use case. Training feature has never being priorities.

**Reply by Ashok-Kumar-dharanikota** [reply]
Okay, thank you for your response 😊. Is there any other library or source you know, that would help me to achieve On-device training ?
